lib/getting-started.tsx

```diff
@@ -1,3 +1,20 @@
 import dayjs from "dayjs";
 
 export const ONBOARDING_INTRODUCED_AT = dayjs("September 1 2021").toISOString();
+
+export const ONBOARDING_NEXT_REDIRECT = {
+  redirect: {
+    permanent: false,
+    destination: "/getting-started",
+  },
+} as const;
+
+export const shouldShowOnboarding = ({
+  completedOnboarding,
+  createdDate,
+}: {
+  completedOnboarding: boolean;
+  createdDate: dayjs.ConfigType;
+}) => {
+  return !completedOnboarding && dayjs(createdDate).isAfter(ONBOARDING_INTRODUCED_AT);
+};
```

Comment:
this could be done so looking at the function we know it has to do with the `User`
```suggestion
import {User} from '@prisma/client';

export const shouldShowOnboarding = (user: Pick<User, 'createdDate' | 'completedOnboarding') => {
  return !completedOnboarding && dayjs(createdDate).isAfter(ONBOARDING_INTRODUCED_AT);
};
```